src/arrdrizmodule.c

```diff
@@ -1146,9 +1146,9 @@ arrxyzero(PyObject *obj, PyObject *args)
   for (j=0; j< imgnum; j++){
     /* compute the delta relative to each source in ref image */
     for (k = 0; k < refnum; k++){
-        dx = *(float *)(PyArray_DATA(imgxy) + j*PyArray_STRIDES(imgxy)[0]) - *(float *)(PyArray_DATA(refxy) + k*PyArray_STRIDES(refxy)[0]);
-        dy = *(float *)(PyArray_DATA(imgxy) + j*PyArray_STRIDES(imgxy)[0]+ PyArray_STRIDES(imgxy)[1]) -
-             *(float *)(PyArray_DATA(refxy) + k*PyArray_STRIDES(refxy)[0]+ PyArray_STRIDES(refxy)[1]);
+        dx = (*(float *)PyArray_DATA(imgxy) + j*(long)PyArray_STRIDES(imgxy)[0]) - (*(float *)PyArray_DATA(refxy) + k*PyArray_STRIDES(refxy)[0]);
```

Comment:
@jhunkeler This does not appear to make sense. It make fixed compiler error but it is just weird.